units/spawnhost_expiration_warning.go

```diff
@@ -0,0 +1,91 @@
+package units
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/alerts"
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/amboy/dependency"
+	"github.com/mongodb/amboy/job"
+	"github.com/mongodb/amboy/registry"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/sometimes"
+	"github.com/pkg/errors"
+)
+
+const spawnhostExpirationWarningsName = "spawnhost-expiration-warnings"
+
+func init() {
+	registry.AddJobType(spawnhostExpirationWarningsName,
+		func() amboy.Job { return makeSpawnhostExpirationWarningsJob() })
+}
+
+type spawnhostExpirationWarningsJob struct {
+	job.Base `bson:"job_base" json:"job_base" yaml:"job_base"`
+}
+
+func makeSpawnhostExpirationWarningsJob() *spawnhostExpirationWarningsJob {
+	j := &spawnhostExpirationWarningsJob{
+		Base: job.Base{
+			JobType: amboy.JobType{
+				Name:    spawnhostExpirationWarningsName,
+				Version: 0,
+			},
+		},
+	}
+	j.SetDependency(dependency.NewAlways())
```

Comment:
Set the time limit for the job

Reply:
This feels unnecessary given that this job only inserts a handful of documents into the db